ehsm_kms_service/kubernetes/ehsm-pv.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume 
+metadata:
+  name: nfspv-ehsm
+spec:
+  capacity:
+    storage: 1Gi 
+  accessModes:
+    - ReadWriteOnce 
+  persistentVolumeReclaimPolicy: Retain 
+  storageClassName: nfs
+  nfs:
+    path: /nfs_ehsm_db

```

Comment:
please keep consistent naming rules (e.g ehsm_xxx)

Reply:
It will be fix.